src/components/Download/DatasetDetails/ExperimentView/index.js

```diff
@@ -1,71 +1,55 @@
 import { useState } from 'react'
 import { Paragraph } from 'grommet'
 import { useDatasetManager } from 'hooks/useDatasetManager'
-import getReadableOptions from 'helpers/getReadableOptions'
 import { OrganismFilter } from './OrganismFilter'
 import { ViewBlock } from './ViewBlock'
 import { ViewBlocks } from '../ViewBlocks'
 
-export const ExperimentView = ({
-  dataset: {
-    data: datasetData,
-    id: datasetId,
-    experiments,
-    quantile_normalize: quantileNormalize
-  },
-  isImmutable
-}) => {
+export const ExperimentView = ({ dataset, isImmutable }) => {
   const { formatSampleMetadata } = useDatasetManager()
+  const defaultOrganismFilterOption = { label: 'All Speciess', value: 'ALL' }
   const defaultValue = 'ALL'
-  const defaultOrganismFilterOption = getReadableOptions([defaultValue])[0]
-  const [organism, setOrganism] = useState(defaultValue)
+  const [organism, setOrganism] = useState(defaultOrganismFilterOption.value)
 
-  if (!datasetData || !Object.keys(datasetData).length) {
+  if (!dataset.data || !Object.keys(dataset.data).length) {
```

Comment:
```suggestion
  const hasExperiments = Object.keys(dataset.data).length > 0
  const filteredExperiments = organism === 'ALL'
    ? dataset.experiments 
    : data.experiments.filter((e) => e.organism_names.includes(organism))
  if (!hasData) {
```

Reply:
I've also applied the same logic to `SpeciesView` to enhance readability by breaking the iteration-related code blocks into helper functions and storing them in constants.

Additionally, I've cleaned up the individual props passed to the sub-component (e.g., `experimentAccessionCode`).